src/Form/Type/UserType.php

```diff
@@ -47,12 +49,27 @@ public function __construct(SkillSetDomain $skillSetDomain)
 
     public function buildForm(FormBuilderInterface $builder, array $options): void
     {
+        /** @var Organization|null $organization */
+        $organization = $builder->getData()->organization;
+
         $occupationChoices = (array) array_combine(self::DEFAULT_OCCUPATIONS, self::DEFAULT_OCCUPATIONS);
         $occupationChoices += ['Autre' => '-'];
         $builder
             ->add('identificationNumber', TextType::class)
             ->add('organization', OrganizationEntityType::class, [
                 'placeholder' => '',
+                'query_builder' => function (OrganizationRepository $repository) use ($organization) {
```

Comment:
you should maybe put this function (or a static one) in the dedicated repository